tide_site.module

```diff
@@ -800,3 +800,49 @@ function tide_site_share_link_token_view_alter(array &$build, EntityInterface $t
     }
   }
 }
+
+/**
+ * Implements hook_tide_card_link_enhancer_undo_transform_alter().
+ *
+ * @see Drupal\tide_landing_page\Plugin\jsonapi\FieldEnhancer\CardLinkEnhancer::doUndoTransform()
+ */
+function tide_site_tide_card_link_enhancer_undo_transform_alter(&$data, &$context) {
```

Comment:
Hey @MdNadimHossain , I think you have to define your own alter API first, otherwise it wouldn't work, an example can be found here https://github.com/dpc-sdp/tide_core/blob/develop/modules/tide_workflow_notification/tide_workflow_notification.api.php.
  correct me if I am wrong @GROwen .

Reply:
@vincent-gao yup it is already defined here - https://github.com/dpc-sdp/tide_landing_page/blob/80911c88485b21796c80eee6c8dd51016720095e/src/Plugin/jsonapi/FieldEnhancer/CardLinkEnhancer.php#L43

Reply:
I meant a `tide_site.api.php` or you can use drupal/symfony event system.

Reply:
Hey @vincent-gao actually it worked. These are the PR environments - 
BE -  https://nginx-php.pr-1056.content-vic.sdp1.sdp.vic.gov.au
FE - https://app.pr-838.vic-gov-au.sdp1.sdp.vic.gov.au